### PR TITLE
CSS Added to the Blog Posts

### PR DESCRIPTION
**Pull request by ollymonger**
Made the blog posts look a bit more interesting by 
formatting them towards the right hand side of the screen.

### CODE REVIEW

File: src/components/App/App.tsx [hunk 1]
```diff
@@ -2,18 +2,23 @@ import React, { Component } from 'react';
 import { BlogPost } from "../BlogPost/BlogPost";
 import { BlogPostRepository } from '../../data/blog-post-repository';
 
+//import "src/style/reactblog.css"; // npm install --save-dev typings-for-css-modules-loader
```
Comment (Jameskmonger) [inline src/components/App/App.tsx]: Don't commit commented-out code to source control. Delete this

File: src/components/App/App.tsx [hunk 2]
```diff
@@ -2,18 +2,23 @@ import React, { Component } from 'react';
...
-    const allBlogPosts = this.blogPostRepository.getAll();
     
+    const allBlogPosts = this.blogPostRepository.getAll();
     return (
-      <div> 
+      <div className="header">
+        
         <h1>Welcome to my blog</h1>
+        <div className="blogPosts">
         <BlogPost blogPost={allBlogPosts[0]} />
```
Comment (Jameskmonger) [inline src/components/App/App.tsx]: This isn't formatted correctly, make sure you have correct indentation

Do CTRL+SHIFT+P and press format document

File: src/components/App/App.tsx [hunk 3]
```diff
@@ -2,18 +2,23 @@ import React, { Component } from 'react';
...
+//import "src/style/reactblog.css"; // npm install --save-dev typings-for-css-modules-loader
+
+
 class App extends Component {
   private blogPostRepository = new BlogPostRepository();
 
   render() {
-    const allBlogPosts = this.blogPostRepository.getAll();
     
+    const allBlogPosts = this.blogPostRepository.getAll();
```
Comment (Jameskmonger) [inline src/components/App/App.tsx]: Why did you move this down a line? You should have a space before your return

```typescript
const allBlogPosts = ...;

return (
```

File: src/style/index.css [hunk 4]
```diff
@@ -12,3 +12,37 @@ code {
...
+}
+
+.header h1{
+  top:10px;
+  border-left:3px solid red;
+  padding-left:15px;
+  position: absolute;
+  right:295px;
+}
+/*
```
Comment (Jameskmonger) [inline src/style/index.css]: Don't commit commented-out code

File: src/style/reactblog.css [hunk 5]
```diff
@@ -0,0 +1,3 @@
+.headerText {
```
Comment (Jameskmonger) [inline src/style/reactblog.css]: Delete this file as it's not used

File: src/style/index.css [hunk 6]
```diff
@@ -12,3 +12,25 @@ code {
...
 }
+
+.header {
+  position: relative;
+  top:35px;
+}
+
+.blogPosts {
+  position: absolute;
+  top:100px;
```
Comment (Jameskmonger) [inline src/style/index.css]: You want to avoid using the following in general as they tend to be bad:

```css
position: absolute;
top: ___;
right: ___;
left: ___;
bottom: ___;
```

The reason that these are bad is that they stop the document from actually flowing like a document. `top` for example will always be at the same distance from the top of the page, so if you add something above it it may not push it down like you expect.

Try to rewrite these CSS styles using `margin` instead of `top` and `right`. It doesn't have to be exactly the same but try to get it looking nice only using margins and padding.

**Comment by ollymonger** [comment]
blog-post-css complete
